data/json/recipes/recipe_food.json

```diff
@@ -1080,6 +1080,22 @@
     "tools": [ [ [ "surface_heat", 5, "LIST" ] ] ],
     "components": [ [ [ "cattail_stalk", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "roasted_cattail_rhizome",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "skills_required": [ "survival", 1 ],
+    "time": "24 m",
+    "autolearn": true,
```

Comment:
I don't think it'd be an autolearn to eat something charred black.

Reply:
It's more like baking potatoes in fire's embers
![image](https://user-images.githubusercontent.com/26396019/101549443-f9eb3980-39ad-11eb-87c8-3cdc70fd44aa.png)

Reply:
So, do you want me to change something?

Reply:
I think ideally this would go in a recipe book or magazine for level one with an auto learn at survival three or four.

Reply:
I set both cooking and survival requirements to 2 and added the recipe to two lowest level survival books.

Reply:
Thanks!